packages/web-components/src/button/button.styles.ts

```diff
@@ -322,7 +323,19 @@ export const styles = css`
     }
 
     :host(:is(:hover, :focus-visible)) {
-      border-color: Highlight;
+      border-color: Highlight !important;
+    }
+
+    :host(${primaryState}:not(:is(:hover, :focus-visible))) {
+      background: Highlight;
```

Comment:
Should this be `background-color` instead so that we don't over styling, the `:host {...}` above only styles `background-color`.